pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1326,7 +1330,9 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		return
 	}
 
+	opComplete := util.OperationCompleteHook(plugin.GetPluginName(), "volume_provision")
```

Comment:
Yeah I think - we can revisit this later, but I do not think it is a problem if API server delays are included in the metric. Similar metrics too include some API server delays too.  My thinking is - it would have been cleaner to do this in `scheduleOperation` because then we don't need to modify each operation individually.